### ISSUE TITLE
JMeter Marketplace

### ISSUE THREAD

**Issue by gbroner2000** [pnopjp/jmeter-plugins#15]
Can this be uploaded to the marketplace so component plugin manager can grab it automatically?

**Comment by kuniteru** [comment]
What is the marketplace?

What is "component plugin pmanaer"?
Is it jmeter-plugins.org?

**Comment by gbroner2000** [comment]
Yes, so jmeter-plugins.org is the plugin centre of Apache Jmeter. It's
important upload ready to use plugins to there.

One of the most common plugins is the plugin manager. It will automatically
add a plug-in to your Jmeter environment if you open a jmeter script that
uses a plugin you don't have installed on your instance. It makes sharing
jmeter scripts much easier.

**Comment by kuniteru** [comment]
Thanks,
As you say, the automatic addition of plugins by the plugin manager is very useful.
I have looked into it and it looks like I can incorporate this plugin into the plugin manager, so I will try.

**Comment by pooja4099** [comment]
@kuniteru many thanks for this plugin firstly. Would you be able to incorporate this into the JMeter plugins repository.
I am using maven (java) to manage plugins and would like maven to grab it automatically. 
thanks

**Comment by JDizzle777** [comment]
Hi @kuniteru, I am unable to find this awesome plugin in the jmeter-plugins.org marketplace. Any luck uploading it? I am trying to incorporate this plugin into an Azure Devops Load Test Pipeline and the only way is to run the JMeter plugin manager command line script to download your plugin

**Comment by JDizzle777** [comment]
Here are some instructions for doing this: https://jmeter-plugins.org/wiki/DeveloperGuide/

**Comment by kuniteru** [comment]
I would also like to list these plugins in the Plugins Manager.
To install this plugins from there, I know that I should register the information for this plugin below.
<https://github.com/undera/jmeter-plugins/blob/master/site/dat/repo/various.json>

But, I am hesitant to implement it due to the following issues.

1. The Developers Guidline at jmeter-plugins.org says to include Unit Tests, but this plugin does not include them.
2. These plugins have separate jar files for each Azure service, but I believe they should be combined into one for distribution in Marketplace.
  It would be difficult to match versions of dependent libraries to achieve this.
3. I'm not sure how to test that what I have registered in verious.json is correctly distributed.

I would be happy to have someone write the Unit Tests code.
